src/System.Runtime.Serialization.Formatters/src/Resources/Strings.resx

```diff
@@ -120,4 +120,14 @@
   <data name="Serialization_OptionalFieldVersionValue" xml:space="preserve">
     <value>Version value must be positive.</value>
   </data>
+  <data name="Serialization_SameNameTwice" xml:space="preserve">
+    <value>Cannot add the same member twice to a SerializationInfo object.</value>
+  </data>
+ <data name="Serialization_NotFound" xml:space="preserve">
+    <value>Member '{0}' was not found.</value>
+  </data>
+  <data name="InvalidOperation_EnumOpCantHappen" xml:space="preserve">
+    <value>Enumeration has either not started or has already finished.</value>
+  </data>
+Serialization_SameNameTwice
```

Comment:
@KrzysztofCwalina did you figure out why this built successfully? It would be good to understand if we have something broken in our tools here.